utils/dev-tool/src/main.rs

```diff
@@ -52,40 +55,26 @@ enum Commands {
 
     /// Make swift jni libs for tests
     MakeSwiftTestLib,
-
-    /// Kill all servers running
-    KillAllServers,
 }
 
 pub struct ToolEnvironment {
     root_dir: PathBuf,
     target_dir: PathBuf,
-    hash_info_dir: PathBuf,
-    server_dbs_dir: PathBuf,
-    commit_hash: String,
+    server_db_dir: PathBuf,
 }
 
 fn main() {
     let root_dir = utils::root_dir();
     let dev_dir = utils::dev_dir();
     let target_dir = utils::target_dir(&dev_dir, &root_dir);
-    let server_dbs_dir = utils::server_dbs_dir(&dev_dir);
-    let hash_info_dir = utils::hash_infos_dir(&dev_dir);
+    let server_db_dir = utils::server_dbs_dir(&dev_dir);
 
-    fs::create_dir_all(&dev_dir).unwrap();
-    fs::create_dir_all(&hash_info_dir).unwrap();
     fs::create_dir_all(&target_dir).unwrap();
-    fs::create_dir_all(&server_dbs_dir).unwrap();
+    fs::create_dir_all(&server_db_dir).unwrap();
 
     env::set_var("CARGO_TARGET_DIR", &target_dir.to_str().unwrap());
```

Comment:
I think you can just set this in `integration.yml` once and then not have this program think about this, ever, right?